S/SDPA/bundled/sdpawrap/sdpawrap.cpp

```diff
@@ -0,0 +1,73 @@
+#include "jlcxx/jlcxx.hpp"
+#include <sdpa_call.h>
+
+namespace jlcxx
+{
+  template<> struct IsBits<SDPA::ConeType> : std::true_type {};
+  template<> struct IsBits<SDPA::PhaseType> : std::true_type {};
+  template<> struct IsBits<SDPA::ParameterType> : std::true_type {};
```

Comment:
Assuming these are enums, the `IsBits` is no longer needed (or even allowed, it is removed from CxxWrap 0.9 and higher).